pkg/build/build.go

```diff
@@ -90,7 +90,7 @@ func Build(manifest model.Manifest) error {
 	}
 
 	if err := GenerateBillOfMaterials(manifest); err != nil {
-		return fmt.Errorf("failed to generate sbom: %v", err)
+		log.Warnf("failed to generate sbom: %v", err)
```

Comment:
Seems like it should be fatal? if we don't want it in some contexts we should explicitly disable?

Reply:
Adding a flag for it instead.